### ISSUE TITLE
Disable telemetry

### ISSUE THREAD

**Issue by dhirschfeld** [conda-forge/dotnet-feedstock#6]
When using the `dotnet` CLI you get an annoying warning:
```
Welcome to .NET 5.0!
---------------------
SDK Version: 5.0.100

Telemetry
---------
The .NET tools collect usage data in order to help us improve your experience. It is collected by Microsoft and shared with the community. You can opt-out of telemetry by setting the DOTNET_
CLI_TELEMETRY_OPTOUT environment variable to '1' or 'true' using your favorite shell.

Read more about .NET CLI Tools telemetry: https://aka.ms/dotnet-cli-telemetry
```

Should we disable telemetry by default?

**Comment by dhirschfeld** [comment]
```
DOTNET_TRY_CLI_TELEMETRY_OPTOUT=true
DOTNET_TRY_SKIP_FIRST_TIME_EXPERIENCE=true
```
See: https://github.com/dotnet/interactive/blob/main/docs/README.md#disclosure

**Comment by dhirschfeld** [comment]
Fixed by #8
